sonic-net/sonic-buildimage
# [Quanta] Update files for Bullseye and kernel 5.10 with enhancements

#### Why I did it
Enhance Quanta files and re-enable the building of Quanta platform modules

#### How I did it
Modify Quanta device and platform files for current master branch and pass PDE tests as many as possible

#### How to verify it
1. Manually update sonic-platform-pde to https://github.com/sonic-net/sonic-platform-pdk-pde/commit/3dcda836ebbdf21b7ed8fc5101cfafd09101477f and modify platfom/test json files in it

2. Build with "INCLUDE_PDE = y" in rules/config for pde-test

3. Verify the show platform { } commands
[show_IX7.log](https://github.com/sonic-net/sonic-buildimage/files/9355744/show_IX7.log)
[show_IX7_BDE.log](https://github.com/sonic-net/sonic-buildimage/files/9355739/show_IX7_BDE.log)
[show_IX8.log](https://github.com/sonic-net/sonic-buildimage/files/9355740/show_IX8.log)
[show_IX8A_BDE.log](https://github.com/sonic-net/sonic-buildimage/files/9355741/show_IX8A_BDE.log)
[show_IX8C.log](https://github.com/sonic-net/sonic-buildimage/files/9355742/show_IX8C.log)
[show_IX9.log](https://github.com/sonic-net/sonic-buildimage/files/9355743/show_IX9.log)

4. Verify with pde-test -v
[pde-test_IX7.log](https://github.com/sonic-net/sonic-buildimage/files/9355750/pde-test_IX7.log)
[pde-test_IX7_BDE.log](https://github.com/sonic-net/sonic-buildimage/files/9355745/pde-test_IX7_BDE.log)
[pde-test_IX8.log](https://github.com/sonic-net/sonic-buildimage/files/9355746/pde-test_IX8.log)
[pde-test_IX8A_BDE.log](https://github.com/sonic-net/sonic-buildimage/files/9355747/pde-test_IX8A_BDE.log)
[pde-test_IX8C.log](https://github.com/sonic-net/sonic-buildimage/files/9355748/pde-test_IX8C.log)
[pde-test_IX9.log](https://github.com/sonic-net/sonic-buildimage/files/9355749/pde-test_IX9.log)

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog
Update files of existing Quanta device/platform

#### Link to config_db schema for YANG module changes

#### A picture of a cute animal (not mandatory but encouraged)
乁( ◔ ౪◔)「
